Resources/Prototypes/_Floof/XenoArch/Effects/normal_effects.yml

```
@@ -105,3 +105,52 @@
       maxAmount: 1
   - type: ReplaceFloorArtifact
     replacement: FloorEighties
+
+- type: artifactEffect
+  id: EffectCashSpawnHigh
+  targetDepth: 2
```

This has a pretty high chance to spawn a large sum of money, so it's best to keep it at a high depth.
```suggestion
  targetDepth: 4
```